projects/igniteui-angular/src/lib/tabs/tabs-group.component.ts

```diff
@@ -160,7 +167,7 @@ export class IgxTabsGroupComponent implements IgxTabsGroupBase, AfterContentInit
      *```
      * @param focusDelay A number representing the expected delay.
      */
-    public select(focusDelay = 200): void {
+    public select(focusDelay = 200, navigateToRoute: boolean = true): void {
```

Comment:
No need to mark this as `Boolean`, as assigning it to `true` by default makes it a Boolean type.

Reply:
Addressed.